### ISSUE TITLE
Support zend-expressive v3

### ISSUE THREAD

**Issue by halfpastfouram** [Lansoweb/basepath#6]
The docs/README.md should include instructions on how to use this library with zend-expressive V3. The example given in README.md does no longer work and will trigger an error.

This can be resolved by using the `doublePassMiddleware` function:

```php
use LosMiddleware\BasePath\BasePath;
use Zend\Diactoros\Response;

/** @var \Zend\Expressive\Application $app */

$app->pipe(\Zend\Stratigility\doublePassMiddleware(new BasePath(), new Response()));
```

**Comment by Lansoweb** [comment]
Hi! Yes, i'm updating most of my libraries, mainly to include psr-15, psr7, expressive 3 and improving code. But thanks to point it out.

**Comment by halfpastfouram** [comment]
Great, I'll keep an eye out for it!

**Comment by Lansoweb** [comment]
Released [2.0.0](https://github.com/Lansoweb/basepath/releases/tag/2.0.0) and it's working with expressive 3. Thanks!
